jvm-packages/xgboost4j-spark-gpu/src/test/java/ml/dmlc/xgboost4j/java/DMatrixTest.java

```diff
@@ -140,43 +153,43 @@ private float[] convertFloatTofloat(Float[]... datas) {
 
   @Test
   public void testMakingDMatrixViaArray() {
-//    Float[][] features1 = {
-//      {1.0f, 12.0f},
-//      {2.0f, 13.0f},
-//      null,
-//      {4.0f, null},
-//      {5.0f, 16.0f}
-//    };
-//
-//    Float[] label1 = {0.0f, 1.0f, 0.0f, 1.0f, 0.0f};
-//
-//    Table X1 = new Table.TestBuilder().column(features1).build();
-//    Table y1 = new Table.TestBuilder().column(label1).build();
-//
-//    ColumnVector t = X1.getColumn(0);
-//    ColumnView cv = t.getChildColumnView(0);
-//    //
-//    System.out.println("----");
-//
-//    Float[][] features2 = {
-//      {6.0f, 17.0f},
-//      {7.0f, 18.0f},
-//    };
-//    Float[] label2 = {0.0f, 1.0f, 0.0f, 1.0f, 0.0f};
-//    Table X2 = new Table.TestBuilder().column(features2).build();
-//    Table y2 = new Table.TestBuilder().column(label2).build();
-//
-//    List<ColumnBatch> tables = new LinkedList<>();
-//    tables.add(new CudfColumnBatch(X1, y1, null, null));
-//    tables.add(new CudfColumnBatch(X2, y2, null, null));
-//
-//    try {
-//      DMatrix dmat = new QuantileDMatrix(tables.iterator(), 0.0f, 8, 1);
-//    } catch (XGBoostError e) {
-//      throw new RuntimeException(e);
-//    }
-//
-//    System.out.println("--------------");
+    //    Float[][] features1 = {
```

Comment:
Do you plan to remove this?

Reply:
Yes, they will be removed in the following PR.